admin_manual/configuration_server/occ_command.rst

```diff
@@ -106,28 +307,88 @@ up the database, so while you can run it manually there usually isn't a need
 to::
   
   $ sudo -u www-data php occ maintenance:repair
-    - Repair mime types  
-    - Repair config
+     - Repair mime types
+ - Repair legacy storages
+ - Repair config
+ - Clear asset cache after upgrade
+     - Asset pipeline disabled -> nothing to do
+ - Generate ETags for file where no ETag is present.
+     - ETags have been fixed for 0 files/folders.
+ - Clean tags and favorites
+     - 0 tags for delete files have been removed.
+     - 0 tag entries for deleted tags have been removed.
+     - 0 tags with no entries have been removed.
+ - Re-enable file app    
  
 User Commands
 -------------
 
-The ``user`` commands reset passwords, display a simple report showing how 
-many users you have, and when a user was last logged in.
+The ``user`` commands create and remove users, reset passwords, display a simple 
+report showing how many users you have, and when a user was last logged in.
+
+You can create a new user with their display name, login name, and any group 
+memberships with the ``user:add`` command. The syntax is::
+
+ user:add [--password-from-env] [--display-name[="..."]] [-g|--group[="..."]] 
+ uid
+
+The ``display-name`` corresponds to the **Full Name** on the Users page in your 
+ownCloud Web UI, and the ``uid`` is their **Username**, which is their 
+login name. This example adds new user Layla Smith, and adds her to the 
+**users** and **db-admins** groups. Any groups that do not exist are created:: 
+ 
+ $ sudo -u www-data php occ user:add --display-name="Layla Smith" 
+   --group="users db-admins" layla
```

Comment:
because group names can also contain spaces, it works a little different.
What you mean is:
```
occ user:add --display-name="Layla Smith"  --group="users" --group="db-admins" layla
or
occ user:add --display-name="Layla Smith"  --group users --group db-admins layla
```

both will output  two lines for "added to group":
```
User "layla" added to group "users"
User "layla" added to group "db-admins"
```
As per the output, what you did added the user to **one** group called `users db-admins`